hera_mc/cm_utils.py

```diff
@@ -199,8 +376,8 @@ def get_astropytime(_date, _time=0):
 
     Returns:  either astropy Time or None
 
-    Parameters:
-    -----------
+    Parameters
+    ----------
     _date:  date in various formats:
```

Comment:
why are your parameters named with an initial underscore? Is it supposed to convey something meaningful to me?

Reply:
I've changed to 'adate' and 'atime' ('date' and 'time' are terrible variable names...)